ci/constraints-test/pandas2-min-install.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.23.0
+pandas==2.0.0rc0
```

Comment:
Would be better if this was 
```
pandas>=2.0.0rc0
```
It would great if this just worked because if we have to add the `--pre` in our install step that might break the other matrix tests.